lib/renderer/security-warnings.js

```diff
@@ -229,23 +229,23 @@ module.exports = {
   },
 
   /**
-   * #10 on the checklist: Do not use blinkFeatures
+   * #10 on the checklist: Do not use enableBlinkFeatures
    *
-   * Logs a warning message about blinkFeatures
+   * Logs a warning message about enableBlinkFeatures
    */
-  warnAboutBlinkFeatures: () => {
+  warnAboutEnableBlinkFeatures: () => {
     const webPreferences = getWebPreferences()
-    if (!webPreferences || !webPreferences.blinkFeatures ||
-        (webPreferences.blinkFeatures.length && webPreferences.blinkFeatures.length === 0)) {
+    if (!webPreferences || !webPreferences.enableBlinkFeatures ||
+        (webPreferences.enableBlinkFeatures.length && webPreferences.enableBlinkFeatures.length === 0)) {
       return
     }
 
-    let warning = 'This renderer process has additional "blinkFeatures" ' +
+    let warning = 'This renderer process has additional "enableBlinkFeatures" ' +
```

Comment:
Can you please change `let` to `const` here?
And the text can be written simpler with a template literal.

(I know it's now a goal of this PR, but let's make this code slightly better.)